modules/libcom/src/error/errSymLib.c

```diff
@@ -56,44 +56,17 @@ extern ERRSYMTAB_ID errSymTbl;
 int errSymBld(void)
```

Comment:
The description of the function can be simplified, just as the function itself has been.